packages/shared-ux/chrome/navigation/src/ui/v2/components/navigation.tsx

```diff
@@ -16,29 +16,46 @@ import React, {
   useContext,
   useRef,
 } from 'react';
+import type { ChromeProjectNavigationNode } from '@kbn/core-chrome-browser';
```

Comment:
Is it OK that the package depends on @kbn/core-chrome-browser ? wouldn't it cause circular deps down the line?

Reply:
I think it is OK. I would expect chrome browser to not depend on almost any packages.
And I don't think the import `type` can create circular dependencies...